ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestService.java

```diff
@@ -122,6 +121,9 @@
 	@Autowired
 	private HazelcastService hazelcastService;
 
+	@Autowired
+	private UncheckedObjectMapper objectMapper;

```

Comment:
If you change `ObjectMapper` extension to another implementation, you should modifed all `@Autowired` filed type.
how about 
```
@Autowired
private ObjectMapper objectMapper;
```

Reply:
That would be nice but, when I autowired as ObjectMapper instance, its methods throws checked exception.